mesheryctl/internal/cli/root/app/offboard.go

```diff
@@ -27,6 +27,10 @@ var offboardCmd = &cobra.Command{
 mesheryctl app offboard -f [filepath]
 	`,
 	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) == 0 {
+			const errMsg = `Usage: mesheryctl app offboard -f [filepath]`
+			return fmt.Errorf("no file path provided \n\n%v", errMsg)
+		}
```

Comment:
Please remove this block because flags are not considered as args, and this command doesn't need any args to work correctly.